autogluon/task/tabular_prediction/predictor.py

```diff
@@ -244,20 +247,62 @@ def leaderboard(self, dataset=None, only_pareto_frontier=False, silent=False):
                     'pred_time_test_marginal': The inference time of the model for the dataset provided, minus the inference time for the model's base models, if it has any.
                         Note that this ignores the time required to load the model into memory when bagging is disabled.
                 If str is passed, `dataset` will be loaded using the str value as the file path.
-            only_pareto_frontier : bool (optional)
+            extra_info : bool, default = False
+                If `True`, will return extra columns with advanced info.
+                This requires additional computation as advanced info data is calculated on demand.
+                Additional output columns when `extra_info=True` include:
+                    'num_features': Number of input features used by the model.
+                    'num_models': Number of models this model represents.
```

Comment:
"Number of models this model represents" may be a bit confusing. Maybe a little clearer in quotations:

`Number of models that actually make up this "model" object.

Reply:
Added